src/components/purchase/directives/purchaseOpenModal/purchaseOpenModal.drv.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * attrs:
+ */
+
+(function (angular) {
+    'use strict';
+
+    angular.module('znk.infra-web-app.purchase').directive('purchaseOpenModal',
```

Comment:
component restrict only to elements, so left it directive with restrict: A
changed name